src/proto_parser.cc

```diff
@@ -164,6 +178,33 @@ LIBRARY_EXPORT int LVGetgRPCAPIVersion(int* version)
     return 0;    
 }
 
+//---------------------------------------------------------------------
+//---------------------------------------------------------------------
+LIBRARY_EXPORT int LVCreateParser(grpc_labview::LVProtoParser** parser)
+{
+    grpc_labview::InitCallbacks();
+
+    *parser = new grpc_labview::LVProtoParser();
+    return 0;
+}
+
+
+//---------------------------------------------------------------------
+//---------------------------------------------------------------------
+LIBRARY_EXPORT int LVAddParserSearchPath(grpc_labview::LVProtoParser* parser, const char* searchPath)
+{
+    parser->AddSearchPath(searchPath);
+    return 0;    
+}
+
+//---------------------------------------------------------------------
+//---------------------------------------------------------------------
+LIBRARY_EXPORT int LVImportProto2(grpc_labview::LVProtoParser* parser, const char* filePath, const char* searchPath)
+{
+    parser->Import(filePath, searchPath);
+    return 0;    
+}
+
 //---------------------------------------------------------------------
 //---------------------------------------------------------------------
 LIBRARY_EXPORT int LVImportProto(const char* filePath, const char* searchPath, grpc_labview::LVProtoParser** parser)
```

Comment:
Would this function still be used anywhere? If not we could probably just replace this with the new implementation instead of creating a new entry point.